content/docs/cml/self-hosted-runners.md

```diff
@@ -83,24 +83,27 @@ you do it your way :)
-| `--cloud-spot`        | Request a spot instance                                                                                       | Boolean                                                                                                        |
-| `--cloud-spot-price`  | Spot max price. If not specified it takes current spot bidding pricing.                                       | default: `-1`                                                                                                  |
-| `-h`                  | Show help                                                                                                     | Boolean                                                                                                        |
+| Argument                 | Description                                                                                                   | Values                                                                                                         |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| `--version`              | Show version number                                                                                           | Boolean                                                                                                        |
+| `--labels`               | Comma delimited runner labels                                                                                 | Default: `cml`                                                                                                 |
+| `--idle-timeout `        | Time in seconds for the runner to be waiting for jobs before shutting down. 0 waits forever.                  | Default: `300`                                                                                                 |
+| `--name`                 | Name displayed in the repo once registered                                                                    | Default: `cml-` followed by a unique identifier, i.e. `cml-cfwj9rrari`                                         |
+| `--single`               | If specified, exit after running a single job.                                                                | Boolean                                                                                                        |
+| `--reuse`                | Don't launch new runners if there is, at least, another one with the same `--name` or containing `--labels`   | Boolean                                                                                                        |
```

Comment:
```suggestion
| `--reuse`                | Don't launch new runners if one already exists with the same name or labels   | Boolean                                                                                                        |
```

Reply:
But it's not clear what the condition is. The existing runner must have the same name AND exact same labels? Just name or some of the labels? Would be good to clarify the exact behavior.

Reply:
🙏  Thanks, @jorgeorpinel! As I tried to express in the original text, there are two disjunctive conditions: for a runner to be reused, it has to either (1) have the same name or (2) contain, at least, the specified labels.

Would the following modification convey the behavior in a clearer way? If not, suggestions are highly welcome.
```suggestion
| `--reuse`                | Don't launch new runners if one already exists with the same name or containing the specified labels   | Boolean                                                                                                        |
```

Reply:
Its better, thanks! But I'm still not getting it 100% with regards to labels. So it's either the exact same name or the exact same labels? If so then my suggestion (previous comment, which I just updated) should be OK, I think.

Reply:
BTW last question on --reuse:

> at least one already exists

Meaning one runner on the same platform (driver)? Or across all drivers? (This may be obvious but I'm not familiar enough with CML yet)

Reply:
@jorgeorpinel, the issue is that, when reusing a runner by labels we don't require an exact match. _Id est:_ if an existing runner [contains](https://en.wikipedia.org/wiki/Subset) all the given labels among many others, we still consider it a match.

Reply:
> _Meaning one runner on the same platform (driver)? Or across all drivers? (This may be obvious but I'm not familiar enough with CML yet)_

For users, the so-called drivers are just the name of their continuous integration vendor, like GitHub Actions, GitLab CI/CD, Azure Pipelines, CircleCI... We only support one of these at a time, so «one runner on the same platform» would be the right meaning.

Reply:
Cools, so maybe
```suggestion
| `--reuse`                | Don't launch a new runner if at an existing one has the same name, or overlapping labels   | Boolean                                                                                                        |
```

Reply:
> _if **at** an_

Is that `at` a residue from the original sentence?

Reply:
```suggestion
| `--reuse`                | Don't launch a new runner if an existing one has the same name, or overlapping labels   | Boolean                                                                                                        |
```

Reply:
> if at an

Yeah that was a typo sry 😅